websites/E/Emby/presence.ts

```diff
@@ -460,24 +513,39 @@ const mediaInfoCache = new Map<string, MediaInfo>();
  */
 async function obtainMediaInfo(itemId: string): Promise<MediaInfo> {
 	if (mediaInfoCache.has(itemId)) return mediaInfoCache.get(itemId);
-
-	let { AccessToken: accessToken } = ApiClient._serverInfo;
+	let ae, accessToken: string;
+	if (!version.includes("4.7")) {
+		ae = ApiClient._serverInfo as ServerLatest;
+		accessToken = ae?.Users?.[1]?.AccessToken;
+	} else {
+		ae = ApiClient._serverInfo as Server;
+		accessToken = ae?.AccessToken;
+	}
```

Comment:
If there's a better solution let me know

Reply:
Maybe add the semver package and use one of their functions if it's a small dependency, on phone can't check rn xd

Reply:
> Also this currently matches on x.4.2 and 4.2.0 idk which you even want

????

Reply:
> > Also this currently matches on x.4.2 and 4.2.0 idk which you even want
> 
> ????

I meant 7 instead of 2..

Reply:
Looking into it